_posts/2023-04-18-Introducing-the-new-OLS.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Introducing the new OLS look
+authors:
+- jilaga
+image: /images/rebranding-Jilaga/2023-04-18-open-by-design.png
+photos:
+  name: photo by Jilaga
+---
+
+Exciting news! We wrapped up our rebranding excercise by **refreshing our logo** and **introducing a new name for our mentoring program**; **Open Seeds**! The new logo reaffirms our core values **Growth**, **Community** and **Nurture**, with a cohesive **colour palette**. We're excited to launch our refreshed brand identity and continue our commitment to advancing open research and scholarship.
+
+> This post is written by Jilaga Nneoma, OLS’s rebranding Outreachy intern, summarising the incredible logo redesign she did over the end of 2022 and first two months of 2023. Now that her internship’s finished, you can contact her via email nneomajilaga@gmail.com . If you’re interested in snagging her incredible talents for your own work, or browse her portfolio [here](https://linktr.ee/jilaga) 
+
+
+## Greetings!
+
+
+It has been quite an exciting journey for us here at Open Life Science (OLS) as we navigate the challenges of expanding our brand identity. As you may have noticed, we recently conducted a rebranding exercise to help us create a new brand identity that aligns with our core values and reflect our growth to include a diverse community of researchers and scientists from a variety of fields.
+
+Initially, we set out to rebrand our name and logo to better represent our mission and core values. However, after careful consideration and feedback from our community, we have decided to keep the acronym OLS as our name. This decision was reached because of the strong association our community members have with the OLS name and what it stands for.
```

Comment:
```suggestion
Initially, we set out to rebrand our name and logo to better represent our mission and core values. However, after careful consideration and feedback from our community, we have decided to **keep the acronym OLS as our name**. This decision was reached because of the strong association our community members have with the OLS name and what it stands for.
```